readme.md

```diff
@@ -7,18 +7,31 @@
 [![slack](https://img.shields.io/badge/-slack-611f69?logo=slack)][slack]
 [![openssf](https://api.securityscorecards.dev/projects/github.com/trunk-io/trunk-action/badge)](https://api.securityscorecards.dev/projects/github.com/trunk-io/trunk-action)
 
+> **Flaky Tests!**
+>
+> Got Flaky Tests? We're building a new tool to handle them and we'd love for you to be included!
```

Comment:
Maybe:

> Got Flaky Tests? We're building a new tool to help identify, track, and quarantine flaky tests and we're looking for partners for our beta program. 
>
>If your team struggles with flaky tests, we would love to work with you.

I'm thinking we can omit the when beta begins and free for 3 month bits. We'll forget to update these, I'm thinking it would be longer lived this way.

Reply:
How about just a 1-liner...

> [!TIP]
>  🎉 *New*: [**Trunk Flaky Tests**](https://trunk.io/flaky-tests) detects, quarantines, and eliminates flaky tests

Reply:
Changed to the 1 liner.